src/opds_dumper.cpp

```diff
@@ -71,7 +72,7 @@ IllustrationInfo getBookIllustrationInfo(const Book& book)
     return illustrations;
 }
 
-std::string fullEntryXML(const Book& book, const std::string& rootLocation)
+std::string fullEntryXML(const Book& book, const std::string& rootLocation, const std::string& bookName)
```

Comment:
We get back to https://github.com/kiwix/libkiwix/pull/830#discussion_r990609558:

> `bookName` is not a good name for this new parameter (taking into account the line `{"name", book.getName()},` below.

The easiest solution for this small PR is to pass into this function the `NameMapper` object instead.

Reply:
What about naming it `content_id` ?